Rakefile

```diff
@@ -79,93 +85,29 @@ task :console do
   IRB.start
 end
 
-desc "Build an archive"
```

Comment:
The :archive task is still there. I wonder if it can be removed since it seems like packaging?

Reply:
I've been using this `archive` task historically to post a zip to AWS for direct download, but that's apparently used very infrequently. I am 👍to deleting this.